.gitignore

```diff
@@ -121,3 +121,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+.vscode/
```

Comment:
<img width="425" alt="Screenshot 2019-04-29 at 10 37 39 PM" src="https://user-images.githubusercontent.com/5113450/56938362-70843d80-6acf-11e9-83de-2c27176bc184.png">

doesn't work as expected...

Reply:
will fix and pr again